### ISSUE TITLE
Google Play Compliance: Precise location supposedly sent off device

### ISSUE THREAD

**Issue by westnordost** [westnordost/StreetComplete#3466]
[Google Play introduced a new data safety policy](https://android-developers.googleblog.com/2021/07/new-google-play-safety-section.html). For app developers, this is basically a form they need to fill out and declare what user data exactly is shared with a third party (sent off the device).

This information will then be shown on the app page on Google Play.

The data types [as documented here](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en-GB#types&zippy=%2Cdata-types) need to be declared.

I filled out the form and declared the following:
- Email address (but only processed ephemerally) - i.e. for OSM account registration
- Personal identifiers - i.e. OSM user name and oauth token

After filling out the form, I got this message:

![image](https://user-images.githubusercontent.com/4661658/139345176-284c964a-3e23-4920-8a4f-2c2ad4f586ee.png)

TLDR: Google Play claims that StreetComplete sends off the precise user's location somewhere which was apparently automatically determined by some AI. Does anyone have any idea what could cause this AI to think that?
